operator/internal/manifests/internal/alerts/prometheus-alerts.yaml

```
@@ -175,3 +175,12 @@ groups:
     for: 15m
     labels:
       severity: warning
+  - alert: lokistackSchemaUpgradesRequired
+    annotations:
+      message: |-
+        Object storage schema needs upgrade.
+      summary: "The applied storage schema config is old and should be upgraded."
+    expr: lokistack_schema_upgrades_required
```

This needs an aggregation to avoid duplicates (for example coming from different instances of the operator):

```suggestion
    expr: sum by(stack_id) (lokistack_schema_upgrades_required) > 0
```

(Note: this is the one of the reasons why you're currently getting two alerts per LokiStack, but it does not solve the underlying issue of the instrumentation code. One remaining issue is, for example, that you will still get alerts for removed LokiStack instances)